src/Writers/LaTexWriter.jl

```diff
@@ -0,0 +1,24 @@
+"""
+Provides the [`render`](@ref) methods to write the documentation as Latex files
+(`MIME"text/latex"`).
+"""
+module LaTexWriter
```

Comment:
Yes, it should be. My ability to sometimes systematically not notice such things never ceases to amaze me -.-